src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/domain/enums/TierMatchCriteria.kt

```diff
@@ -1,23 +0,0 @@
-package uk.gov.justice.digital.hmpps.hmppstier.domain.enums
```

Comment:
This was an invented requirement, I think we're much better off either saving the scores used to calculate the tier with the score or doing nothing